frontend/apps/web/components/ListBox/ListBox.tsx

```diff
@@ -39,16 +39,19 @@ export default function ListBox<TData>(props: Props<TData>): ReactElement {
   return (
     <div
       className={cn(
-        'h-[164px] relative w-full rounded-md border border-gray-300 dark:border-gray-700 ',
+        'h-[164px]  rounded-md border border-gray-300 dark:border-gray-700 w-[350px]',
```

Comment:
I think the `relative` is pretty important here for the pinning of the virtual rows.